Die Zeit.js

```diff
@@ -1,167 +1,140 @@
 {
 	"translatorID": "312bbb0e-bfb6-4563-a33c-085445d391ed",
 	"label": "Die Zeit",
-	"creator": "Martin Meyerhoff",
+	"creator": "Philipp Zumstein",
 	"target": "^https?://www\\.zeit\\.de/",
-	"minVersion": "1.0",
+	"minVersion": "3.0",
 	"maxVersion": "",
 	"priority": 100,
 	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcsibv",
-	"lastUpdated": "2014-04-03 16:49:40"
+	"lastUpdated": "2015-11-04 07:45:52"
 }
 
 /*
-Die Zeit Translator
-Copyright (C) 2011 Martin Meyerhoff
+	***** BEGIN LICENSE BLOCK *****
 
-This program is free software: you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation, either version 3 of the License, or
-(at your option) any later version.
+	Copyright © 2015 Philipp Zumstein
 
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-GNU General Public License for more details.
+	This file is part of Zotero.
 
-You should have received a copy of the GNU General Public License
-along with this program. If not, see <http://www.gnu.org/licenses/>.
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
 */
 
 /*
 This translator works only partially, because zeit.de uses some strange javascript that makes 
-processDocuments return an error. If I just call scrape(doc, url) on a single document, it works. 
-The way the translator is programmed now, it only works if JavaScript is turned off in the browser.
-
-Try it out here:
-http://www.zeit.de/wirtschaft/2011-03/schnappauf-ruecktritt-stuttgart
-http://www.zeit.de/online/2009/12/arbeitsrecht-urlaub
-http://www.zeit.de/suche/index?q=Krise
-http://www.zeit.de/2009/11/
+processDocuments return an error. If I just call scrape(doc, url) on a single document, it works.
+The way the translator is programmed now, it only works for multiples if JavaScript is turned off in the browser.
+For example at 
+   http://www.zeit.de/suche/index?q=Krise
+only the first reference can be scraped.
```

Comment:
The multiples were not working before and they are still not working. Any idea how I could fix that?